Sources/SnapshotPreferences/ExpansionPreference.swift

```diff
@@ -19,7 +19,26 @@ struct ExpansionPreferenceKey: PreferenceKey {
 }
 
 extension View {
-  public func emergeExpansion(_ enabled: Bool?) -> some View {
-    preference(key: ExpansionPreferenceKey.self, value: enabled)
-  }
+    /// Applies an expansion effect to the view's snapshot.
+    ///
+    /// Use this method to control the emerge expansion effect on a view. When enabled,
+    /// the engine will attempt to expand the view's first scrollview to the maximum height possible.
```

Comment:
"the engine"? Not sure what that is, maybe just say "When enabled the view's first scrollview will be expanded to show all content in the snapshot"